services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/presigner/DefaultS3Presigner.java

```diff
@@ -98,9 +98,13 @@
 @SdkInternalApi
 public final class DefaultS3Presigner extends DefaultSdkPresigner implements S3Presigner {
     private static final AwsS3V4Signer DEFAULT_SIGNER = AwsS3V4Signer.create();
+    private static final S3Configuration DEFAULT_S3_CONFIGURATION = S3Configuration.builder()
+            .checksumValidationEnabled(false)
+            .build();
```

Comment:
Why is disabling checksum validation required?

Reply:
This is just carrying forward the existing default this class was using on line 292. To be honest, I'm not real sure why it is set that way to begin with.